README.md

```diff
@@ -13,13 +15,49 @@ This Rails project:
 - has scaffolding, standard CRUD actions and shopping cart (without purchasing part)
 - includes: products, orders, line items, users, store, log in, log out, forms, sessions, admin, unit tests, integration tests, order notifications, etc.
 
-## To run the application:
+## To run the web app that is in a Docker container and Deployed on AWS
+
+1. Create an EC2 instance on AWS with an Ubuntu image and at least a t2.micro size.
+
+2. Create a security rule for TCP to use port 48019 and use that with the instance.
+
+3. ssh to the EC2 instance, e.g.,
+   `ssh -i <full path to .pem key> ubuntu@<IPaddress>`
+
+4. Create a `.env` file on EC2 instance with environment variables similar to the example below. Assign values to the MySQL root user and the MySQL user along with **passwords**, which **should NOT be shown to anyone**.
+
+  ```
+    MYSQL_ROOT_USER=root
+    MYSQL_ROOT_PASSWORD=therootuserpassword
+    MYSQL_USER=yourmysqluser
+    MYSQL_PASSWORD=yourmysqluserpassword
+    MYSQL_DATABASE=rails_buy_stuff_development
+    PORT_ASSIGNED_ON_HOST_FOR_DB=48018
+    PORT_ASSIGNED_ON_HOST_FOR_WEB_APP=48019
+    PORT_RAILS_WEB_SERVER_DEFAULT=3000
+    PORT_MYSQL_SERVER_DEFAULT=3306
+  ```
+
+5. Run: `main.sh` Type .pem key and IP address.
+
+6. Go to a browser and see the running app. Use the IP address of the instance and port, e.g., `<http://54.190.12.61:48017>`
+
+To locally run the app that is in a Docker container using Docker Compose
+Clone the repo: git clone git@github.com:kimlew/rails_see_stuff.git
+
+Change to project directory with: cd rails_see_stuff
+
+Run: docker compose up
+
+Go to the browser address: <http://localhost:48017/>
```

Comment:
Do you still need these lines?  You describe how to run locally in the next section, with more complete instructions.

Reply:
Confirmed deleted.